lib/adapters/translate_encoder.js

```diff
@@ -30,6 +30,8 @@ var encoder = processor({
         return null;
       case 'boolean':
         return val;
+      case 'uuid':
```

Comment:
Your call. it's a fixed width type according to the spec so I thought i belonged here but I'm fine adjusting that.